### PR TITLE
Update for 8.0b0 release.

### PR DESCRIPTION
**Pull request by hjoliver**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hjoliver** [comment]
@conda-forge-admin, please rerender

**Comment by hjoliver** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,52 +1,47 @@
...
   version: {{ version }}
 
 build:
-  number: 2
+  number: 0
   # Skip as Cylc requires Linux
   skip: True  # [py<37 or not linux]
 
+# Note: to pin build numbers, if necessary, e.g.:
+#    - cylc-flow {{ version }} *_3
```
Comment (kinow) [inline recipe/meta.yaml:13]: :pray: I hope we never need it again, but good idea to leave it as a note!

**Review by kinow — APPROVED** [review]
Looks good to me!!! :clap:

**Comment by hjoliver** [comment]
Built and tested locally.

**Review by oliver-sanders — APPROVED** [review]
Hub issues spotted in testing are specific to Jupyterhub forge build and our local setup (phew).

Looks sane, merge if you want to push ahead with the discrete meta-package for now (rather than updating docs), happy either way.

**Comment by hjoliver** [comment]
Thanks @oliver-sanders - I'll merge this *and* update the docs, because it makes thing better for those who do (based on our previous advice and/or intuition) try `conda install cylc`.  And as discussed we can try to come up with the best solution later, before 8.0.

**Pull request merged**